src/api/ledger/parse/pathfind.js

```diff
@@ -2,13 +2,23 @@
 'use strict';
 const parseAmount = require('./amount');
 
+// test fixture has an example response where source_amount is an IOU without
```

Comment:
Are you sure? I didn't get an answer to that when I asked in the development channel yesterday.